salt/utils/event.py

```diff
@@ -586,6 +591,9 @@ def __del__(self):
             self.destroy()
         except Exception as ex:
             log.debug(ex)
+            # as this method can be called when logging is already teared down
+            # just try to stdout it
+            traceback.print_exc(file=sys.stderr)
```

Comment:
well, for that special case,  it just worked as soon as i didnt not used neither of logging or print(), but directly the stderr stream.